istio-control/istio-autoinject/templates/serviceaccount.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.clusterResources }}
```

Comment:
Is there any use case to enable this rather than installing with the CRDs?

Reply:
Old installer and most of our tests are generating those cluster resources - not ready to disable it yet. 

However in future I think it is clearly better to separate the cluster resources ( and the high-priv credentials needed to deploy them ) from the actual deployment+config. A CI/CD may push upgrades to various components - but it would only need istio-system ( or istio-telemetry ) permissions, not cluster-wide rights.